### ISSUE TITLE
Using bundler >=2 breaks CI

### ISSUE THREAD

**Issue by llekn** [platanus/potassium#204]
Newly created projects use bundler 2.0.1 in their `Gemfile` which breaks the CI with the following error:

```
You must use Bundler 2 or greater with this lockfile
```

**Comment by llekn** [comment]
It can be fixed modifying the `docker-ci` files:

`Dockerfile.ci`:

```dockerfile
FROM platanus/ruby:2.5

RUN gem install bundler --version 2.0.1

RUN mkdir /app
WORKDIR /app

ADD . /app
```

`docker-compose.ci.yml`:

```YAML
---
version: '3.4'
services:
  test:
    build:
      context: "."
      dockerfile: Dockerfile.ci
    volumes:
    - "test_data:/usr/local/bundle"
    environment:
      RACK_ENV: test
      DB_USER: postgres
      DB_HOST: postgresql
      DB_PORT: '5432'
    links:
    - postgresql
  postgresql:
    image: postgres:11.3
    environment:
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: ''
volumes:
  test_data:
```

**Comment by difernandez** [comment]
After applying those changes another error ocurred with webpacker

```
The git source git://github.com/rails/webpacker.git is not yet checked out. Please run 'bundle install' before trying to start your application
```

It was solved by removing the `, github: 'rails/webpacker'` from `gem 'webpacker'` in the Gemfile

**Comment by difernandez** [comment]
Anothe problem: webpacker wasn't installed from the beginning due to a package being incompatible with node version 9:

![image](https://user-images.githubusercontent.com/12057523/62222809-f8553e80-b381-11e9-8d96-a2cbc13704c7.png)

Maybe we should ad a `.node-version` file in the recipe

**Comment by difernandez** [comment]
The last two comments were extracted to a separate issue: #217

**Comment by difernandez** [comment]
@blackjid this should be solved with platanus/docker-ruby#2. I tested it in a project with bundler 2 (and removing the fix it had) and [CI ran without any problems](https://circleci.com/gh/platanus/big-pictures/70), I'm closing it.
